make/apis.mk

```
@@ -22,7 +22,7 @@ PROVIDER_API_VERSION_capx := v1beta1
 # Instead. sync their APIs into the api/external directory.
 .PHONY: apis.sync
 apis.sync: ## Syncs infrastructure providers' APIs
-apis.sync: $(addprefix api.sync.,capa capd caaph capx) go-fix.api
+apis.sync: $(addprefix api.sync.,capa capd caaph capx) go-fix.api mod-tidy.api
```

Actually, let's run `mod-tidy` as this may also tidy dependencies in root module.

```suggestion
apis.sync: $(addprefix api.sync.,capa capd caaph capx) go-fix.api mod-tidy
```